minio/minio
# Pass Console Subpath envrionment variable

Right now console supports being served on a subpath via `CONSOLE_SUBPATH` but the MinIO server is not setting this value, so this PR introduces `MINIO_CONSOLE_SUBPATH` to configure this variable in turn.

To test:

Run MinIO Like
```shell
MINIO_BROWSER_REDIRECT_URL="http://localhost:8000/console/quack" CI=true ./minio server /tmp/dskx{1...4} --address :9000 --console-address :9091
```

start nginx with the following config
```
events { worker_connections 1024; }

http {

server {
    listen 8000;

    location /console/quack/ {
        rewrite   ^/console/quack/(.*) /$1 break;
        proxy_pass http://localhost:9091;
    }
}

}
```

Visit http://localhost:8000/console/quack/ or http://localhost:9000 (which should redirect properly)

Known issue: login assets don't load properly, but everything else works fine.

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>
